lib/saml/user.rb

```diff
@@ -22,23 +26,39 @@ def to_hash
     end
 
     def authn_context
```

Comment:
Changing this breaks the `decorator_constant` method, as well as the use of `user.authn_context` in the sessions controller to build the multifactor URL. In the interest of minimal changes I'd leave this method alone and create a private method that gets the value from the saml_response, to be used only in the `log_warnings_to_sentry` method.

Reply:
By "leave alone" I mean put it back to what it was before this PR.

Reply:
oh... you mean for user.authn_context it's no longer going to be nil, it will be `http://identiymangagement/loa/1` hmmm you might right about that actually.

Reply:
And the dslogon_multifactor and mhv_multifactor cases will no longer match.

Reply:
well, those could be handled a little differently since they're based on the values on current_user. could make sure that user.authn_context is still set to nil if its not mhv or dslogon... but i think that's going too far down an untested rabbit hole... i'll redo this.